src/interface/IHooks.sol

```diff
@@ -18,13 +33,44 @@ interface IHooks {
     );
     event SetPerformanceFee(uint256 oldFee, uint256 newFee);
```

Comment:
this interface should not have anything about fees, fee functions fee events

That's specific to the Fee Hook

This interface should only be a IHook Interface

Can define a separate interface for those things and have the concrete implementation use both